api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -900,7 +913,7 @@ private void enableDispensing() {
                 sessionLocationHasTag("Dispensing Location")));
 
         extensions.add(visitAction(Extensions.DISPENSE_MEDICATION_VISIT_ACTION,
-                "dispensing.app.label",
+                "Receta",
```

Comment:
Just create a message override for `dispensing.app.label`.

Reply:
Agree, yes, we don't want to hard-code a name here....

Reply:
Ping @MiguelAHPpih , this is actually important

Reply:
@brandones I'm not using that method.

Reply:
Using what method? This dispensing app is used by several sites; we can't have it show up as "Receta" in Haiti.

Reply:
Oh i think this was from when i tried to use the dispensing component, I'll leave it as it was.

Reply:
Perfect, thanks